src/test/java/com/crud/tasks/mapper/TrelloMapperTestSuite.java

```diff
@@ -0,0 +1,143 @@
+package com.crud.tasks.mapper;
+
+import com.crud.tasks.domain.*;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TrelloMapperTestSuite {
+
+    @InjectMocks
+    TrelloMapper trelloMapper;
+
+    @Test
+    public void testMapToCard() {
+        //Given
+        TrelloCardDto trelloCardDto = new TrelloCardDto("Card 1", "Taking orders", "Three", "14");
+        //When
+        TrelloCard trelloCard = trelloMapper.mapToCard(trelloCardDto);
+        //Then
+        Assert.assertNotEquals(trelloCardDto, trelloCard);
```

Comment:
Trochę ta asercja wydaje mi się dziwna - skoro obiekty są różnych typów (i nie są ze sobą w relacji dziedziczenia), to z definicji nie powinny być równe...